lib/Sema/CSSimplify.cpp

```diff
@@ -1003,11 +1024,18 @@ getCompletionArgIndex(ASTNode anchor, ConstraintSystem &CS) {
     auto idx = llvm::find_if(elems, [&](Expr *elem) {
       return CS.containsCodeCompletionLoc(elem);
     });
-    if (idx != elems.end())
-      return std::distance(elems.begin(), idx);
+    if (idx != elems.end()) {
+      unsigned completionArgIdx = std::distance(elems.begin(), idx);
+      auto firstTrailingIdx =
+          TE->getUnlabeledTrailingClosureIndexOfPackedArgument();
+      return CompletionArgInfo{ completionArgIdx, firstTrailingIdx };
```

Comment:
I bit of repetitive code here which would be great to get into a separate closure.